<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1,0">
      <item>
       <layout class="QVBoxLayout" name="layoutPlotAI1"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layoutPlotAI2"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layoutPlotUI1"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_saveAll">
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_loadAll">
          <property name="text">
           <string>Загрузить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_clearAll">
          <property name="text">
           <string>Очистить всё</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="log"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройка</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionConfigure"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConfigure">
   <property name="text">
    <string>Настройка</string>
   </property>
   <property name="toolTip">
    <string>COM-port setting</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Подключить</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Отключить</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
